packages/Liquid-Core.package/LiquidAnswerPanel.class/instance/question..st

```diff
@@ -0,0 +1,14 @@
+accessing
+question: aQuestion
+
+	question := aQuestion.
+	selectedChoices := Set new.
+	(question type = 'singleChoice')
+	ifTrue: [self maxNumberOfChoices: 1.
+				self answer: LiquidSingleChoiceAnswer new.]
```

Comment:
since this is dependent on `question type`, maybe we could have `question buildEmptyAnswer` builder method?

Reply:
Seems like a good suggestion to me, I will include this in the next commit.